src/SlmLocale/Strategy/UriPathStrategy.php

```diff
@@ -196,7 +201,12 @@ public function assemble(LocaleEvent $event)
         // Remove first part
         array_shift($parts);
 
-        $path = $base . '/' . $locale . '/' . implode('/', $parts);
+        $locale .= '/';
+        if ($locale === $this->default) {
```

Comment:
is this correct? to be true default value should be set to something like `en/` with `/` at the end.

Reply:
50/50
in fallback it should be with `/`, because i removed it some lines down.